packages/ni.measurementlink.discovery.v1.proto/docs/intro.inc

```diff
@@ -0,0 +1,26 @@
+Introduction
+============
+
+About
+-----
+
+The ``ni.measurementlink.discovery.v1`` Python package contains generated Python interfaces for proto files in the ``ni-apis`` repo.
+Additionally, this package contains methods for converting between Python and protobuf types.
```

Comment:
Remove this line. This is copy pasta from ni.protobuf.types. We should not have handwritten code for protobuf conversion. However at some point we want to think about including hand written discovery client in this package.